initial-data-load/src/mapLayers/layersGeoDataUpload.js

```diff
@@ -105,6 +105,9 @@ const formatLayerGeoData = async (data, country) => {
     } else {
       logger.info(`Layer ${data.name} has corrupted or missing geojson file.`);
     }
+  } else if (!data.geoDataFilename && !data.geoDataUrl && data.apiUrl && data.collectionName) {
```

Comment:
would not one if before all this at L76 if(data.collectionName) be enough? It could cover new collection in each case and be easier to understand